### PR TITLE
types: add team types

### PR DESCRIPTION
**Pull request by itohatweb**
These types are required by the oauth types which are required for the Ready type

### CODE REVIEW

File: src/types/users/partial_user.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { User } from "./user.ts";
+
```
Comment (ayntee) [inline src/types/users/partial_user.ts:2]: Missing link

Comment (itohatweb) [reply src/types/users/partial_user.ts:2]: link for?

Comment (ayntee) [reply src/types/users/partial_user.ts:2]: Any documentation link regarding the fields of the partial user object.

Comment (ayntee) [reply src/types/users/partial_user.ts:2]: In that case, you should just do `Partial<User>`

Comment (itohatweb) [reply src/types/users/partial_user.ts:2]: to be exact there is this but this is only for the team member object
image

Comment (ayntee) [reply src/types/users/partial_user.ts:2]: In that case, we should just go with your solution :stuck_out_tongue_closed_eyes:

**Review by ayntee — APPROVED** [review]

**Pull request merged**